tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "extends": "./src/custom/tsconfig-paths-custom.json",
```

Comment:
linter?

Reply:
Yep.
For some reason, local formatting always applies one format, CI applies another.

Have to fix this, eventually...